contributor/05-benefits-of-contribution-article.asciidoc

```diff
@@ -14,72 +14,72 @@ their purview?
+hard to meassure, but has been reported both in InnerSource and in Open Source
 projects alike: people participate because they find work on those projects
-personally fullfilling and fun. Most likely, the aspect of being in a position
+personally fulfilling and fun. Most likely, the aspect of being in a position
 to truly self select tasks to work on does play an important role.
 This self selection typically also leads to host projects being very welcoming
-and supportive as those are key to keeping motivation in contributors up.
+and supportive in their effort to keep contributors motivated.
 
 === Team level motivation
 
-Remember that annoying bug that's suddenly fixed upstream? Why should your
-team spend the extra effort to contribute that fix to the upstream project?
+Remember that annoying bug that's been suddenly fixed upstream? Why should your
+team spend an extra effort to contribute that fix to the upstream project?
 
-For one it means that maintenance cost and time is now with the upstream
+For one, it means that maintenance cost and time is now with the upstream
 project.  For every new release it's on them instead of on your team to make sure it
 keeps working with your modifications and requirements.
 
 Having team members work as active contributors in projects your team depends on
-means that likely they will get to have a say in project direction and timelines,
+means that they may get to have a say in the project direction and timelines,
 which can be beneficial for your team.
 
 By using InnerSource teams can establish a middle path between "be independent
-and build your own" (including any number of new bugs that you own) and "save
+and build your own" (including any number of new bugs that you introduce) and "save
 time and money by relying on existing implementations" (at the cost of creating
-long term dependencies that can only be influenced in a limited way). That way
-balancing reimplementing versus reusing becomes easier.
+long term dependencies which can only be influenced in a limited way). That way
+balancing reimplementing versus reuse becomes easier.
 
 === Company motivation
 
 Remember that functionality that's specific to your company domain - but that
 is maintained in mutiple implementations across the entire company? What if
-there were a way to avoid a dozen buggy implementations and merge them into a
-shared asset with a development process attached that avoids the usual
+there were a way to avoid a dozen buggy versions and merge them into a
+shared asset? What if the development process for this shared asset ran without the usual
 drain of energy that central dependencies bring to the table? Many open source
-projects are being used by a huge number of players - some of which participate
+projects are being used by a huge number of players - some of who participate
 in their design and development. Encouraging cross team collarboration in InnerSource
```

Comment:
Not a change that you introduced, but still want to suggest fixing this typo.

```suggestion
 in their design and development. Encouraging cross team collaboration in InnerSource
```

Reply:
oops thank you!